### DISCUSSION TITLE
B 站首页新增全屏预览功能

### DISCUSSION THREAD

**Discussion by timongh** [the1812/Bilibili-Evolved#3814] [分享 / Share]
B站新功能：鼠标悬浮在首页视频上会开启全屏预览

已发现需要更改的组件：自定义顶栏（需要调整 z-index）
可能需要跟进的组件：极简首页、清爽首页
可能的需求：禁止全屏预览（或通过按钮开启全屏预览）

全屏预览效果演示：

https://user-images.githubusercontent.com/46739861/201522943-42421f58-8a6e-4f90-a219-9624ecad5a9e.mp4

**Comment by happyTonakai** [comment]
很需要禁止全屏预览，这哪个大聪明产品想出来的
